core/domain/skill_services_test.py

```diff
@@ -394,97 +359,155 @@ def test_get_filtered_skill_summaries(self):
         self.assertEqual(augmented_skill_summaries[1].id, self.SKILL_ID2)
 
     def test_filter_skills_by_status(self):
```

Comment:
Here and below: Have separate tests with clear test names which explain what exactly you are trying to test in each test.

Alternatively, put single-line comments above each test "block" within a test to explain what that test block is trying to measure/catch.

Reply:
Done, but I am confused as to where to add the comments in tests(was unable to understand that by lookin in the codebase)